### ISSUE TITLE
Add public api to autocomplete to check whether it is open

### ISSUE THREAD

**Issue by jelbourn** [angular/material2#5881]
There is no public api users can consume that tells them whether an autocomplete is open on `MdAutocomplete`. We should add one.

**Comment by leocaseiro** [comment]
Hi @jelbourn, I'd like to work on that feature.

We have some emitters for the select component such as `onOpen` and `onClose`.
I wonder if that would be the best approach or should we export public the property `showPanel`?

Thanks

**Comment by jelbourn** [comment]
@leocaseiro I was just imagining an `isOpen` property on `MdAutocomplete`

**Comment by carlos-** [comment]
This can already be done with the `panelOpen` property on `MdAutocompleteTrigger`. I updated [this Plunker](https://plnkr.co/edit/gPp7voxywnsGjVQC8rXs?p=preview) to prove myself it works.

@jelbourn, do you want to offer it directly in the MdAutocomplete, or should this be fine?

**Comment by willshowell** [comment]
@carlos- Can't speak for the original intent, but I think this would be nice

```html
<md-autocomplete #auto="mdAutocomplete">
  <md-option *ngFor="let state of filteredStates | async" [value]="state">
    {{ state }}
  </md-option>
</md-autocomplete>

isOpen: {{auto.isOpen}}
```

since it would eliminate the need for this:

```ts
@ViewChild(MdAutocompleteTrigger) trigger: MdAutocompleteTrigger;

isOpen() {
  return this.trigger.panelOpen
}
```

**Comment by carlos-** [comment]
@willshowell, I totally agree. On the other hand, the same could apply to `openPanel` and `closePanel`. Right?

**Comment by kkotak** [comment]
@carlos- Thanks for the tip on using MdAutocompleteTrigger. I am using it successfully for one Autocomplete on the page. How can I use it with multiple Autocompletes on the same page?

**Comment by josephperrott** [comment]
Fixed by #7149
